### DISCUSSION TITLE
Prioritize certain operations over others

### DISCUSSION THREAD

**Discussion by VinceG** [meilisearch/product#93] [Feedback & Feature Proposal]
**Is your feature request related to a problem? Please describe.**
While indexing 10K records we delete a document, that delete operation will now be in queue until all the 10K records are indexed. we should probably be able to specify the priority of these operations or at least allow to perform certain operations immediately instead of queueing them up. 

**Describe the solution you'd like**
Allow to specify the priority of indexing operations, for example DocumentDelete will be performed right away before indexing documents

**Describe alternatives you've considered**

**Additional context**
it creates a large delay to remove a document which in some cases is important than indexing other documents.

**Comment by curquiza** [comment]
Hello @VinceG! Thanks for your report!
I don't think this is in our current roadmap. However, we have a public roadmap where our user can share their ideas and their needs. It would be awesome if you can add your idea so that other users will upvote it if they need it as well.
https://roadmap.meilisearch.com/tabs/5-ideas

**Comment by VinceG** [comment]
@curquiza Thank You! Submitted.
